src/test/docker/bionic/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fluxrm/flux-core:bionic-v0.28.0
+FROM fluxrm/flux-core:bionic-v0.33.0
```

Comment:
Just a thought - if you just use `fluxrm/flux-core:bionic`, then builds will always use flux-core@master. For the time being, this may be easier than trying to track the latest official release.

Reply:
Maybe I am doing something wrong... just using `fluxrm/flux-core:bionic` is what I used to do in this repo, but for some reason it builds with an old version of flux-core, so I switched to manually specifying the flux-core version. For example, this is my output of `./configure` when using `FROM fluxrm/flux-core:bionic` and launching an interactive Docker container with the following command:

```console
$ ./src/test/docker/docker-run-checks.sh --no-cache --no-home -i bionic -I --
```

```
  flux-accounting version 0.12.0-24-g5eeb904
  Prefix...........: /usr
  Debug Build......: 
  C Compiler.......: gcc
  C++ Compiler.....: g++
  CFLAGS...........: -g -O2
  CPPFLAGS.......... 
  CXXFLAGS.......... -g -O2
  FLUX.............: /usr/bin/flux
  FLUX_VERSION.....: 0.24.0-8-g7c7c85128
  FLUX_CORE_CFLAGS.: 
  FLUX_CORE_LIBS...: -lflux-core
  LIBFLUX_VERSION..: 0.24.0-8-g7c7c85128
  FLUX_PREFIX......: /usr
  LDFLAGS..........: 
  LIBS.............: 
  Linker...........: /usr/bin/ld -m elf_x86_64
  PYTHON...........: /usr/bin/python3
```

Reply:
By default `docker` doesn't grab the latest `flux-core:bionic` if one already exists locally.
I think if you first run `docker pull flux-core:bionic`, then the latest built docker image will be fetched before you run `docker-run-checks.sh`. Can you try that and let me know how it goes?

Reply:
Oh, I see! 🤦‍♂️ You're 100% right. I need to pull down the latest versions to my machine locally before building. I always thought it tried to pull a latest version automatically when running `docker-run-checks.sh`.

Running `docker pull fluxrm/flux-core:bionic` _and then_ running `docker-run-checks.sh` built with the latest version of flux-core. I'll edit this commit with a useful message accordingly. Sorry for the noise!